packages/x509/x509.0.6.3/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+name: "x509"
+maintainer: [
+  "Hannes Mehnert <hannes@mehnert.org>" "David Kaloper <david@numm.org>"
+]
+authors: [
+  "David Kaloper <david@numm.org>" "Hannes Mehnert <hannes@mehnert.org>"
+]
+license: "BSD2"
+tags: "org:mirage"
+homepage: "https://github.com/mirleft/ocaml-x509"
+doc: "https://mirleft.github.io/ocaml-x509/doc"
+bug-reports: "https://github.com/mirleft/ocaml-x509/issues"
+depends: [
+  "ocaml" {>= "4.04.2"}
+  "dune" {build & >= "1.2"}
+  "ppx_sexp_conv"
```

Comment:
```suggestion
  "ppx_sexp_conv" {>="v0.11.0"}
```

This constraint ensures that older non-dune ppx_sexp_convs wont be picked up (which can happen on older compilers)